src/test/java/pl/rejmanbeata/bakery/jpa_repository/TestHelper.java

```diff
@@ -0,0 +1,26 @@
+package pl.rejmanbeata.bakery.jpa_repository;
+
+import pl.rejmanbeata.bakery.database.AddressEntity;
+import pl.rejmanbeata.bakery.database.OrderEntity;
+import pl.rejmanbeata.bakery.database.ProductEntity;
+
+import java.util.Random;
+
+public class TestHelper {
```

Comment:
as we talked on today's meeting: I would suggest to rename this class to `EntitiesFactory` and put all similar methods here and name them in one way. For example `generateAddress` or `generateEmployeeWithRole("assistant")`. :)